### ISSUE TITLE
[FEATURE] Funder specific components

### ISSUE THREAD

**Issue by vicradon** [BuildForSDG/Team-083-Frontend#32]
Write all components a funder would expect to see

**Comment by princeadeyeye** [comment]
what do you mean by the funder components?
is it
 1. Dashboard
2. Auth page
3. Donate Fund
4. View All Donations
5. View All SMEs..

Please be specific..

**Comment by princeadeyeye** [comment]
is the funder using the same dashboard/auth pages as sme

**Comment by vicradon** [comment]
Yes. We check if the user-type is of type **funder** and load the components needed by that user. So that we can easily reuse components.

We also need admin components. The logic would be the same as before.

**Comment by vicradon** [comment]
@princeadeyeye 
I have created the structure of this flow on the project.

It's on [this branch](https://github.com/BuildForSDG/Team-083-Frontend/tree/ft-admin-components)

Please, pull and start adding more functionality.

You don't have to work on admin-components. You can work on funder components
